de.dlr.sc.virsat.server.test/src/de/dlr/sc/virsat/server/dataaccess/TransactionalJsonProviderTest.java

```diff
@@ -63,7 +63,6 @@
 
 	@Before
 	public void setUp() throws Exception {
-		super.setUp();
```

Comment:
This is a bit strange. If we dont need the setup from the super class, then we are we extending AServerRepositoryTest?

Reply:
You are right that is strange. I looked into it and it called the `setUp()` from the `AGitAndJettyServerTest`, that `AServerRepositoryTest `extends, that I deleted in this PR. The problem causing this misunderstanding was wrong naming of the   junit setup/teardown methods in `AServerRepositoryTest`, so I will rename them to make it clear.